### PR TITLE
Initial navigation setup

### PR DESCRIPTION
**Pull request by shripada**
Initial navigation setup

### CODE REVIEW

File: CleanCity/App.js [hunk 1]
```diff
@@ -5,23 +5,13 @@
...
 
 import React, { Component } from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
-
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' + 'Cmd+D or shake for dev menu',
-  android: 'Double tap R on your keyboard to reload,\n' + 'Shake or press menu button for dev menu',
-});
+import { StyleSheet, View } from 'react-native';
+import RootNavigation from './app/config/routes';
```
Comment (KeerthiSP) [inline CleanCity/App.js]: Please ensure that header is added

Comment (shripada) [reply CleanCity/App.js]: Yes, I will adresss that.

**Review by KeerthiSP — CHANGES_REQUESTED** [review]
Please document this file

**Review by KeerthiSP — APPROVED** [review]

**Comment by shahisthasheik** [comment]
looks good merging it

**Pull request merged**